reduce_test.go

```diff
@@ -0,0 +1,163 @@
+package godash_test
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/thecasualcoder/godash"
+	"strconv"
+	"testing"
+)
+
+func TestReduce(t *testing.T) {
```

Comment:
Could you add an example for a test of Word Count

Reply:
Added